UserSampleId/app/src/main/java/com/swrve/usersampleid/MainActivity.java

```diff
@@ -0,0 +1,54 @@
+package com.swrve.usersampleid;
+
+import android.os.Bundle;
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+
+import com.swrve.sdk.SwrveSDK;
+
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        SwrveSDK.onCreate(this);
+
+        // Use the SwrveSampleIdUtils to genereate a number between 1-100 for the
+        // userId. This function should be called as soon as possible after the
+        // SwrveSDK.OnCreate(this) method.
+        SwrveSampleIdUtils.sendSampleIdForUser();
```

Comment:
Unfortunately can't be called in the Application class because the Swrve SDK is not fully initialized until the onCreate() method of the first Activity.